src/pages/resources/mainnet/upgrades/v1.0.mdx

```diff
@@ -0,0 +1,76 @@
+# Mainnet upgrade: v1.0.1
+
+import { Callout } from '/src/components/callout'
+
+## About the `axelard` v1.0.1 upgrade
```

Comment:
Can you mention each binary? axelard, vald and tofnd. We listed this out explicitly in previous instructions